### PR TITLE
WEBEU2 - Preprocessing I - Nabeelah Yousuph 🇳🇬

### PR DESCRIPTION
**Pull request by NabeelahY**
Creating a personal portfolio

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,13 @@
 <!doctype html>
 
 <html lang="en">
+
 <head>
   <meta charset="utf-8">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (maximesalomon) [inline index.html:7]: Good job adding this!

File: index.html [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
   <title>My Resume</title>
-
+  <link href="https://fonts.googleapis.com/css?family=Dosis" rel="stylesheet">
```
Comment (maximesalomon) [inline index.html:9]: How could you do this with LESS?

File: index.html [hunk 3]
```diff
@@ -14,6 +16,84 @@
...
+        <p>Connect With Me:</p>
+        <div>
+          <a href="https://github.com/NabeelahY/" target="_blank"><i class="fa fa-github"></i></a>
+          <a href="https://linkedin.com/in/nabeelah-yousuph/" target="_blank"><i class="fa fa-linkedin"></i></a>
+        </div>
+        <div>
+          <p> &copy 2019 Copyright. Nabeelah Yousuph</p>
+        </div>
+      </div>
+
```
Comment (maximesalomon) [inline index.html:94]: Great HTML structure 👍

File: less/main.less [hunk 4]
```diff
@@ -0,0 +1,213 @@
+@mainColour : #fe5f55;
```
Comment (maximesalomon) [inline less/main.less:1]: Great job using a separate LESS file! You could even create other LESS files for your variables or for different sections

File: less/index.less [hunk 5]
```diff
@@ -54,6 +54,10 @@ table {
...
 html, body {
-    height: 100%;
+	height: 100%;
+	font-family: 'Dosis', sans-serif;
 }
-
+:root{
+	font-size: 100%;
+}
+@import "main.less";
```
Comment (maximesalomon) [inline less/index.less:63]: Yes :)

File: less/main.less [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+@backgroundColour: rgb(231, 228, 228);
+@sizeSmall: 0.5rem;
```
Comment (maximesalomon) [inline less/main.less:3]: Good job with the variables

File: less/main.less [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+    
+            &:hover {
+                color: @mainColour;
+            }
+        }
+    
+    }
+}
+
+/* LOGO */
```
Comment (maximesalomon) [inline less/main.less:52]: good job with the comments!

**Review by maximesalomon — COMMENTED** [review]
Great job on this Nabeelah 👍 

- You created a main.less file and use import ✅
- You used variables to set fonts & colors ✅
- You found a good balance with your nesting ✅
 - You used media queries ✅
- You did not use mixins ⚠️

Make sure you read the README fully before starting your projects

Great job using the commit guidelines, love your small bullet points description

You got this 💪

File: index.html (hunk 1 above)
Comment (NabeelahY) [reply index.html:7]: Thank you!

File: index.html (hunk 2 above)
Comment (NabeelahY) [reply index.html:9]: @import url ('google font link')

File: index.html (hunk 3 above)
Comment (NabeelahY) [reply index.html:94]: Thank you

**Comment by NabeelahY** [comment]
Thanks for the feedback

**Pull request merged**